client/cypress/integration/InfoBase/route_tests.spec.js

```diff
@@ -247,38 +247,37 @@ const route_load_tests_config = [
 
 describe("Route tests", () => {
   route_load_tests_config.map((routes) => {
-    describe(`${routes.name} route`, () => {
+    describe(`${routes.name}`, () => {
       routes.test_on.map((app) => {
-        it(`Tested on ${app}`, () => {
-          cy.on("fail", (e, test) => {
-            if (routes.expect_to_fail) {
-              console.log("Test expected to fail.");
-            } else {
-              throw e;
-            }
-          });
-
+        it(`Tested on index-${app}.html#${routes.route}`, () => {
           cy.visit(
             `http://localhost:8080/build/InfoBase/index-${app}.html#${routes.route}`
           );
-          if (!routes.skipAxe) {
-            if (app == "eng" || app == "fra") {
-              cy.injectAxe();
-              cy.get(".leaf-spinner__inner-circle", { timeout: 10000 }).should(
-                "not.exist"
-              );
-              cy.checkA11y(
-                null,
-                { includedImpacts: ["critical"] },
-                cy.terminalLog,
-                false
-              );
-            }
+
+          if (!routes.skipAxe && (app == "eng" || app == "fra")) {
+            cy.injectAxe();
+            cy.get(".leaf-spinner__inner-circle", { timeout: 10000 }).should(
+              "not.exist"
+            );
+            cy.checkA11y(
+              null,
+              { includedImpacts: ["critical"] },
+              cy.terminalLog,
+              false
+            );
```

Comment:
Is tit necessary to inject axe, make the leaf spinner check, and then do the a11y check in that order? Just wondering if the common `cy.get...` between the two branches of this `if` could be moved up right after the `cy.visit` so that
  1. the operation isn't duplicated and we don't risk having them go out of sync
  2. so that the `cy.get` part runs under the same conditions whether or not the axe checks also runs after (don't know if the `cy.injectAxe()` call could impact the spinner test, but seems worth being cautious)